code/game/machinery/_machinery.dm

```diff
@@ -955,11 +955,38 @@
 	return TRUE
 
 /obj/machinery/proc/display_parts(mob/user)
-	. = list()
-	. += span_notice("It contains the following parts:")
+	var/init_value=0
```

Comment:
```suggestion
	var/init_value = 0
```

It's not obvious what this refers to by the way

Reply:
It's just a value used to initialize the starting count for a particular part/stack. I put a comment now specifying that